rocketpy/tools.py

```diff
@@ -374,126 +373,127 @@ def inverted_haversine(lat0, lon0, distance, bearing, earth_radius=6.3781e6):
-    except KeyError:
-        print("No apogee data found. Skipping apogee ellipses.")
-        apogee_x = np.array([])
-        apogee_y = np.array([])
-    try:
-        impact_x = np.array(results["x_impact"])
-        impact_y = np.array(results["y_impact"])
-    except KeyError:
-        print("No impact data found. Skipping impact ellipses.")
-        impact_x = np.array([])
-        impact_y = np.array([])
-
-    # Define function to calculate Eigenvalues
-    def eigsorted(cov):
-        # Calculate eigenvalues and eigenvectors
-        vals, vecs = np.linalg.eigh(cov)
-        # Order eigenvalues and eigenvectors in descending order
-        order = vals.argsort()[::-1]
-        return vals[order], vecs[:, order]
-
-    def calculate_ellipses(list_x, list_y):
-        # Calculate covariance matrix
-        cov = np.cov(list_x, list_y)
-        # Calculate eigenvalues and eigenvectors
-        vals, vecs = eigsorted(cov)
-        # Calculate ellipse angle and width/height
-        theta = np.degrees(np.arctan2(*vecs[:, 0][::-1]))
-        w, h = 2 * np.sqrt(vals)
-        return theta, w, h
-
-    def create_ellipse_objects(x, y, n, w, h, theta, rgb):
-        """Create a list of matplotlib.patches.Ellipse objects.
-
-        Parameters
-        ----------
-        x : list or np.array
-            List of x coordinates.
-        y : list or np.array
-            List of y coordinates.
-        n : int
-            Number of ellipses to create. It represents the number of confidence
-            intervals to be used. For example, n=3 will create 3 ellipses with
-            1, 2 and 3 standard deviations.
-        w : float
-            Width of the ellipse.
-        h : float
-            Height of the ellipse.
-        theta : float
-            Angle of the ellipse.
-        rgb : tuple
-            Tuple containing the color of the ellipse in RGB format. For example,
-            (0, 0, 1) will create a blue ellipse.
-
-        Returns
-        -------
-        list
-            List of matplotlib.patches.Ellipse objects.
-        """
-        ell_list = [None] * n
-        for j in range(n):
-            ell = Ellipse(
-                xy=(np.mean(x), np.mean(y)),
-                width=w,
-                height=h,
-                angle=theta,
-                color="black",
-            )
-            ell.set_facecolor(rgb)
-            ell_list[j] = ell
-        return ell_list
+    ell = Ellipse(
+        xy=(np.mean(x), np.mean(y)),
+        width=w,
+        height=h,
+        angle=theta,
+        color="black",
+    )
+    ell.set_facecolor(rgb)
+    ell.set_alpha(opacity)
+    return ell
+
+
+def generate_monte_carlo_ellipses(
+    apogee_x=np.array([]),
+    apogee_y=np.array([]),
+    impact_x=np.array([]),
+    impact_y=np.array([]),
+    n_apogee=[1, 2, 3],
+    n_impact=[1, 2, 3],
+    apogee_rgb=(0, 1, 0),
+    impact_rgb=(0, 0, 1),
+    opacity=0.2,
+):  # pylint: disable=dangerous-default-value
```

Comment:
Is it alright to simply disable this warning instead of changing the defaults? I understand that is not very used outside of internal code, I am asking just to confirm the reasoning.

Reply:
Yes it is more than OK,
I just don`t even know why I didnt disable this pylint rule yet. 
Leave it there.